server/routes/manage-csips/tests.cy.ts

```diff
@@ -11,28 +11,49 @@ context('test /manage-csips', () => {
     cy.task('reset')
     cy.task('stubSignIn')
     cy.task('stubComponents')
+
+    // All stubs should be available to all tests to ensure API query params are correctly constructed
+    cy.task('stubSearchCsipRecords')
+    cy.task('stubSearchCsipRecordsClosed')
+    cy.task('stubSearchCsipRecordsOpen')
+    cy.task('stubSearchCsipRecordsPlans')
+    cy.task('stubSearchCsipRecordsReferrals')
   })
 
   it('tests all use cases', () => {
-    cy.task('stubSearchCsipRecords')
     navigateToTestPage()
     cy.url().should('to.match', /\/manage-csips$/)
     checkAxeAccessibility()
 
     checkSortingAccessibility()
 
-    cy.findAllByRole('heading', { name: /View and manage CSIPs/i }).should('be.visible')
+    cy.findAllByRole('heading', { name: /CSIP Caseload/i }).should('be.visible')
+
+    cy.get('.light-box > p > a').should('have.length', 2)
+    cy.get('.light-box > p > a').eq(0).should('have.text', 'Plans only')
+    cy.get('.light-box > p > a').eq(1).should('have.text', 'Referrals in progress only')
 
     getFilterButton().should('be.visible')
     getClearLink().should('be.visible')
     getQueryInput().should('be.visible').and('have.value', 'A1234CD')
     getStatusSelect().should('be.visible').and('have.value', '')
 
+    // All statuses are represented in the table
+    cy.get('.govuk-table__body > tr > td > strong').eq(0).should('have.text', 'CSIP closed')
+    cy.get('.govuk-table__body > tr > td > strong').eq(1).should('have.text', 'CSIP open')
+    cy.get('.govuk-table__body > tr > td > strong').eq(2).should('have.text', 'Awaiting decision')
+    cy.get('.govuk-table__body > tr > td > strong').eq(3).should('have.text', 'Plan pending')
+    cy.get('.govuk-table__body > tr > td > strong').eq(4).should('have.text', 'Investigation pending')
+    cy.get('.govuk-table__body > tr > td > strong').eq(5).should('have.text', 'Referral submitted')
+    cy.get('.govuk-table__body > tr > td > strong').eq(6).should('have.text', 'Referral pending')
+    cy.get('.govuk-table__body > tr > td > strong').eq(7).should('have.text', 'No further action')
+    cy.get('.govuk-table__body > tr > td > strong').eq(8).should('have.text', 'Support outside of CSIP')
+
```

Comment:
I'd also confirm all the expected columns are visible and vice versa that those that shouldn't be visible are not for the variants of this page